parsers/EIA.py

```diff
@@ -338,16 +338,32 @@
     "US-MIDW-MISO->US-SE-SOCO": {"US-MIDW-MISO->US-SE-AEC"},
 }
 
+# # IMPORTANT: the order of the fields must match the order of the fields in the database
```

Comment:
Why is this important?

Reply:
it's not, copied from the power breakdown struct. removed